.github/CONTRIBUTING.md

```diff
@@ -1,134 +1,94 @@
 
-If you require more time to provide a time frame for completion, you are allowed to take up to five business days to think about it.
+The main way to contribute to the project is by working on an issue.
+Look through the **Prioritized** column in the [Community Backlog][] for issues that are available for you to work on.
 
-If you can't get back to us by that time, we understand!
-As a community developer, you are helping us out in addition to your regular job.
-We will un-assign you from this issue, but please feel free to contribute to another issue.
+_Do not attempt to work on something unrelated to the issues inside the **Prioritized** column._
 
-#### Step 4: Work on the issue
+If you see an issue you would like to work on in the **Prioritized** column, notify a Community Maintainer in the issue comments (by using @) to let them know you are interested.
+The Community Maintainer will follow up with you to make sure you understand the scope of the changes being asked for in the issue.
 
-After you provide an estimate, the issue is now "in progress".
+After the Community Maintainer assigns the issue to you, it is moved to the **In Progress** column of the [Community Backlog][] to prevent others from picking up the issue.
 
-If you need more time to work on the issue, please contact us as soon as possible.
-We may request an update on your progress, but we are willing to accommodate.
+#### Working on the solution
 
-If the deadline you provided to us passes and we have not heard from you, we will wait one week before un-assigning you from the issue.
+As a contributer, you should familiarize yourself with the project's [coding standards and conventions][].
 
-#### Step 5: Create a pull request
+The issue will provide you with guidance for what we think the solution should look like.
+If you are unsure about anything, reach out to anyone in the Community Advisory Board and we will provide more details.
 
-When you finish working on an issue, create a pull request with the issue number included in the title or body.
-This starts the (brief) code review process.
+To get your contribution accepted, you must sign [Adobe's Contributor License Agreement (CLA)](https://opensource.adobe.com/cla.html).
+You only need to sign the CLA once.
 
-After we accept and merge your contribution, you become an official contributor!
-Official contributors are invited to our backlog grooming sessions and have direct influence over the product roadmap.
+#### PR review process
 
-We hope this guide paints a clear picture of your duties and expectations in the contribution process. Thank you in advance for helping with our research projects!
+After you submit your PR, the Community Advisory Board will assign one or more reviewers to look over your solution and ensure quality and adherence to standards.
+As the Directly Responsible Individual for the issue, you are expected to address all feedback before the issue is sent to our QA and merged.
```

Comment:
I would add something here that makes this sound less overwhelming to new developers.

> If at any point in the review cycle you have commitments preventing you from addressing feedback, or if there's anything suggested you're not understanding, please let us know; we're happy to pair program or complete those changes for you to push PRs across the finish line.